pages/api/twitter/signup.js

```diff
@@ -85,18 +85,12 @@ export default async (req, res) => {
       .send({ decorator: "SERVER_CREATE_USER_BUCKET_INIT_FAILURE", error: true });
   }
 
-  const photo = await SlateManager.getRandomSlateElementURL({
-    id: Environment.AVATAR_SLATE_ID,
-    fallback:
-      "https://slate.textile.io/ipfs/bafkreick3nscgixwfpq736forz7kzxvvhuej6kszevpsgmcubyhsx2pf7i",
-  });
-
   const user = await Data.createUser({
     username: newUsername,
     email: newEmail,
     twitterId: twitterUser.id_str,
     data: {
-      photo,
+      photo: "",
```

Comment:
I think it's best if we leave it out, rather than setting photo to an empty string